nnvm/python/nnvm/frontend/keras.py

```diff
@@ -666,12 +680,13 @@ def from_keras(model):
                 zip_node = zip(node.node_indices, node.tensor_indices, node.inbound_layers)
                 for n_idx, t_idx, layer in zip_node:
                     if isinstance(layer, keras.engine.InputLayer):
-                        sym = symtab.get_var(layer.name, must_contain=True)
+                        sym_name = layer.name
                     else:
                         sym_name = layer.name + ':' + str(n_idx) + ':' + str(t_idx)
-                        sym = symtab.get_var(sym_name, must_contain=True)
-                    insym.append(sym)
-
+                    # In some models, sym_name may not be available in inbound_nodes
```

Comment:
Can you share an example model where `sim_name` is not available in inbound_nodes?

Reply:
This change ignores the encoder outputs in the decoder model, and uses zeros as an initial state instead.  It looks wrong to me.

The root cause is that you are processing layers which are included in the imported model but not relevant to the current model.  You can skip such layers with the below code.
```
    if not model._node_key(keras_layer, node_idx) in model._network_nodes:
        continue
````
Note that model._network_nodes contains keys of all nodes relevant to the current model.